content/docs/security/secure-connections.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Secure connections
+---
+
+Neon requires that all connections use SSL/TLS encryption to ensure that data sent over the Internet cannot be viewed or manipulated by third parties. Connections that do not use SSL/TLS are rejected. 
+
+Neon enforces SSL/TLS connections by enabling  `ssl` in PostgreSQL and using only `hostssl` records in the `pg_hba.conf` client authentication configuration file. A `hostssl` record matches connection attempts made using TCP/IP, but only if the connection is made with SSL/TLS encryption. For related information, see [The pg_hba.conf File](https://www.postgresql.org/docs/current/auth-pg-hba-conf.html), in the _PostgreSQL documentation_.
```

Comment:
Let's remove this paragraph, see
https://neondb.slack.com/archives/C036U0GRMRB/p1673325779724649?thread_ts=1673301585.286539&cid=C036U0GRMRB
we can add 'behaving like usual PostgreSQL with hostssl records in pg_hba.conf file only' to the last sentence.

Reply:
Thank you. Revised as suggested.